server/routes/api.js

```diff
@@ -9,10 +10,14 @@ router.get("/flags", flagsController.getAllFlags);
 // router.get("/flags", validateSDKKey, flagsController.getAllFlags);
 router.get("/flags/:id", flagsController.getFlag);
 
-router.post("/flags", validateNewFlag, flagsController.createFlag);
+router.get("/stream", streamController.handleNewConnection)
 
-router.put("/flags/:id", flagsController.editFlag);
+router.get("/status", streamController.status);
```

Comment:
Can you remind me what the GET /status endpoint is for?

Reply:
Get status is just to let us know how many people are currently connected in the SSE connection.